### PR TITLE
Milestone 1 : Awesome books: plain JavaScript with objects 

### PR DESCRIPTION
**Pull request by michael-duke**
# Awesome Books : Plain JS with objects

In this milestone @aceDavon  and @michael-duke  have:
- Understood different ways to create objects in JavaScript.
- Created and accessed properties and methods of JavaScript objects.
- Followed best practices for HTML, CSS by implementing basic UI feels.
- Followed MVC structure and best JavaScript best practices.
- Disabled `"no-use-before-define": ["error", {"functions": false}]`, to allow the use of hoisting.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  const bookAuthor = document.getElementById('author');
+  const { value: author } = bookAuthor;
+
+  const newBook = new CreateBook();
+  newBook(id, title, author);
+  renderBooks(bookCollection);
+
+  bookTitle.value = '';
+  bookAuthor.value = '';
+}
```
Comment (t-yanick) [inline js/main.js]: - [ ] Hello team, your are doing a great job so far. Please when I add a new book(inputting the value of title and author and clicking on the `add button`), nothing is displayed on the page, but when I refresh the page, the books are displayed  on page but the title and author values are undefined, see screenshot below and kindly fix the issue. 

udefinedAwesome

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  this.author = author;
+  bookCollection.push(this);
+  saveCollection();
+}
+
+// Deletes a Book
+function removeBook(bookId) {
+  bookCollection = bookCollection.filter(({ id }) => id !== bookId);
+  saveCollection();
+}
```
Comment (t-yanick) [inline js/main.js:53]: - [x] Please when I remove any book(`when  I click on the remove button`) from that page,  I deletes all the books, please kindly fix the issue.Please make sure that when you delete a particular, only that book should be deleted and the localStorage updated according accordingly.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi Team, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean/Organize code,
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 2 above)
Comment (michael-duke) [reply js/main.js:53]: Thank you for the feedback and note we have made the required changes 😄 Happy Coding @t-yanick

**Review by FrederickMih — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by michael-duke** [comment]
Thanks @FrederickMih, and Happy Coding

**Pull request merged**